### PR TITLE
Plone 5.1

### PR DESCRIPTION
**Pull request by Nachtalb**
Plone 5.1 after upgrade from Plone 4.3
---
image

Plone 4.3
---
image

### CODE REVIEW

**Comment by Nachtalb** [comment]
@jone can you add @4teamwork/onegovbox to this project, please.

**Comment by jone** [comment]
@Nachtalb done. btw, Mäthu can do this too as he is org owner.

File: ftw/faqblock/profiles/default-plone5/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<metadata>
+    <dependencies>
+        <dependency>profile-ftw.upgrade:default</dependency>
```
Comment (busykoala) [inline ftw/faqblock/profiles/default-plone5/metadata.xml]: Don't we need ftw.simplelayout for this block?

File: ftw/faqblock/profiles/default-plone5/registry.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<registry>
+
+    <records prefix="plone.bundles/ftw-faqblock-resources" interface='Products.CMFPlone.interfaces.IBundleRegistry'>
+        <value key="enabled">True</value>
+        <value key="depends">plone</value>
+        <value key="stub_js_modules">jquery</value>
+        <value key="compile">False</value>
+        <value key="jscompilation">++resource++ftw.faqblock/js/faqblock-compiled.js</value>
+        <value key="merge_with">default</value>
+        <value key="last_compilation">2020-06-26 00:42:00</value>
```
Comment (busykoala) [inline ftw/faqblock/profiles/default-plone5/registry.xml:11]: I don't really care but was corrected here once... So if you care you could adapt the comp. date.

File: ftw/faqblock/upgrades/20200723135311_plone_5_upgrade/registry.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+<registry>
+
+    <records prefix="plone.bundles/ftw-faqblock-resources" interface='Products.CMFPlone.interfaces.IBundleRegistry'>
+        <value key="enabled">True</value>
+        <value key="depends">plone</value>
+        <value key="stub_js_modules">jquery</value>
+        <value key="compile">False</value>
+        <value key="jscompilation">++resource++ftw.faqblock/js/faqblock-compiled.js</value>
+        <value key="merge_with">default</value>
+        <value key="last_compilation">2020-06-26 00:42:00</value>
```
Comment (busykoala) [inline ftw/faqblock/upgrades/20200723135311_plone_5_upgrade/registry.xml:11]: ...same here. If you care, you might want to correct the date.

File: test-plone-5.1.x.cfg [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/test-plone-5.1.x.cfg
+    sources.cfg
+
+package-name = ftw.faqblock
+
+versions = versions
+
+
+[versions]
+setuptools = <45a
```
Comment (busykoala) [inline test-plone-5.1.x.cfg:12]: I know we did that in some pkgs, but in my opinion that should go to ftw-buildouts if possible.

File: test-plone-5.1.x.cfg [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+zc.buildout = 2.13.3
```
Comment (busykoala) [inline test-plone-5.1.x.cfg:13]: Why do you pin zc.buildout?

**Review by busykoala — APPROVED** [review]
Looks good. I've added some comments to which you could react or just leave it as it is...

File: test-plone-5.1.x.cfg (hunk 4 above)
Comment (Nachtalb) [reply test-plone-5.1.x.cfg:12]: We need this because v45 and later is not py2 compatible.

File: test-plone-5.1.x.cfg (hunk 5 above)
Comment (Nachtalb) [reply test-plone-5.1.x.cfg:13]: Why exactly it was pinned idk but it is already for 4.3 and I just copied over and updated the versions. ¯\_(ツ)_/¯

File: ftw/faqblock/profiles/default-plone5/metadata.xml (hunk 1 above)
Comment (Nachtalb) [reply ftw/faqblock/profiles/default-plone5/metadata.xml]: Nice catch this was already wrong from the start 👍

File: ftw/faqblock/profiles/default-plone5/registry.xml (hunk 2 above)
Comment (Nachtalb) [reply ftw/faqblock/profiles/default-plone5/registry.xml:11]: The date doesn't really matter and if you want to see when it was changed just use `git blame`
ʕノ•ᴥ•ʔノ ︵ ┻━┻

File: ftw/faqblock/upgrades/20200723135311_plone_5_upgrade/registry.xml (hunk 3 above)
Comment (Nachtalb) [reply ftw/faqblock/upgrades/20200723135311_plone_5_upgrade/registry.xml:11]: Same as above

File: test-plone-5.1.x.cfg (hunk 4 above)
Comment (busykoala) [reply test-plone-5.1.x.cfg:12]: I know, but that doesn't mean that we have to do it here. All our packages depend on that being pinned. Why not put it into ftw.buildouts?

**Review by maethu — APPROVED** [review]

**Pull request merged**